### PR TITLE
Write CodeDeploy documentation

### PR DESCRIPTION
**Pull request by flomotlik**

### CODE REVIEW

File: _posts/docker/deployments/2015-09-01-aws.md [hunk 1]
```diff
@@ -220,6 +220,19 @@ If you are using more than once instance for your application you need to add at
 }
 ```
 
+## Deploying to CodeDeploy
+
+To deploy your application to AWS CodeDeploy you can use the integrated `codeship_aws codedeploy_deploy` command. It will zip your application code, upload it to s3 and start a new deployment on CodeDeploy. You can take a look at the [full script](https://github.com/codeship-library/aws-deployment/blob/master/scripts/codeship_aws_codedeploy_deploy) in the [codeship-library/aws-deployment](https://github.com/codeship-library/aws-deployment) repository on Github.
```
Comment (mlocher) [inline _posts/docker/deployments/2015-09-01-aws.md]: ... upload it to _S3_ and start ...

**Comment by mlocher** [comment]
@flomotlik one minor change, is this finished otherwise?

**Comment by flomotlik** [comment]
@mlocher fixed that one. Waiting for the PR on the repo to be reviewed and finished, but other than that we're done with this one.

**Comment by mlocher** [comment]
ok, so :shipit: from me, @flomotlik merge when ready :)

**Pull request merged**